show/main.py

```diff
@@ -778,6 +778,42 @@ def services():
         else:
                 break
 
+@cli.command()
+def aaa():
+    """Show AAA configuration in ConfigDb"""
+    config_db = ConfigDBConnector()
+    config_db.connect()
+    data = config_db.get_table('AAA')
+    output = ''
+    if data != {}:
+        for row in data:
+            entry = data[row]
+            for key in entry:
+                output += ('AAA %s %s %s\n' % (row, key, str(entry[key])))
+
```

Comment:
can we print out the default values for fallback and fallthough?

Reply:
OK, I'll change it.